### DISCUSSION TITLE
Android App?

### DISCUSSION THREAD

**Discussion by delize** [aaronpk/Overland-iOS#172] [General]
Just generally curious, is there any intent to revive the Android app? 

The readme / docs for the iOS app of overland say "[Grab the app here](https://github.com/OpenHumans/Overland_android)", but, [that repo has not been updated in 3 years](https://github.com/OpenHumans/Overland_android/commits/dev/), and the [Google Play Store App is no longer available](https://play.google.com/store/apps/details?id=com.overlandbound.ob1&hl=en). 

Is there an alternative now a days if Overland won't support Android going forward?

**Comment by aaronpk** [comment]
hmm, I'm not sure what happened to the Android app! I wasn't involved in the development other than them asking if it was okay to copy the style of my iPhone app. I'd suggest opening an issue on their repo to find out more.
